src/packages/index.gleam

```diff
@@ -141,6 +141,25 @@ create table if not exists hidden_packages (
   name text primary key
 ) strict;
 
+create view if not exists non_retired_packages as
+  -- A package is retired if its latest release is retired
```

Comment:
A package is retired if _all_ its releases are retired. Something like this:

```sql
create view non_retired_packages as
select
  p.id
, p.name
, p.description
, p.docs_url
, p.links
, p.updated_in_hex_at
from packages p
where p.id in (
  select distinct r.package_id
  from releases r
  where r.id is null or r.retirement_reason is null
);
```

Reply:
oops! Fixed, thanks for the explanation